hunsri/EwMM-ProjectCS
# ft/PauseMenuColFix

Fixed:
- collision problem on pause menu scripts

Test:
- open the train scene where the pause menu is integrated
- try to get a NPC to a button which could be triggered by projectiles like `Back to Menu`
- it should not be triggered by NPCs anymore
